spec/factories/lti_services.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :lti_service do
+    lti_deployment { association(:lti_deployment) }
```

Comment:
you can just write `lti_deployment`, by default the association will be generated automatically